docs/releases/1.8.txt

```diff
@@ -73,7 +73,8 @@ New PostgreSQL specific functionality
 
 Django now has a module with extensions for PostgreSQL specific features, such
 as :class:`~django.contrib.postgres.fields.ArrayField`,
-:class:`~django.contrib.postgres.fields.HStoreField`, :ref:`range-fields`, and
+:class:`~django.contrib.postgres.fields.HStoreField`, :ref:`range-fields`,
+:class:`~django.contrib.postgres.widgets.RangeWidget`, and
```

Comment:
This is out of place -- form fields aren't even mentioned in the release notes and widgets are even more "obscure". I agree with Simon that a note isn't needed here.